src/rsa_primitive_verify_tests.txt

```diff
@@ -0,0 +1,5 @@
+# Big-endian public key parts for signature_rsa_example_public_key.der,
+# encoded in hex.
+
+n = CEA80475324C1DC8347827818DA58BAC069D3419C614A6EA1AC6A3B510DCD72CC516954905E9FEF908D45E13006ADF27D467A7D83C111D1A5DF15EF293771AEFB920032A5BB989F8E4F5E1B05093D3F130F984C07A772A3683F4DC6FB28A96815B32123CCDD13954F19D5B8B24A103E771A34C328755C65ED64E1924FFD04D30B2142CC262F6E0048FEF6DBC652F21479EA1C4B1D66D28F4D46EF7185E390CBFA2E02380582F3188BB94EBBF05D31487A09AFF01FCBB4CD4BFD1F0A833B38C11813C84360BB53C7D4481031C40BAD8713BB6B835CB08098ED15BA31EE4BA728A8C8E10F7294E1B4163B7AEE57277BFD881A6F9D43E02C6925AA3A043FB7FB78D
```

Comment:
On Sat, Aug 27, 2016 at 3:07 AM, Dirkjan Ochtman <notifications@github.com>
wrote:

> Is it possible you made an off-by-one error here in suggesting putting e
> around 2 ** 34? According to AGL
> <https://www.imperialviolet.org/2012/03/16/rsae.html>, it seems like that
> maximum e should be 2 ** 32 + 1, which has a length of 33 bits. As such,
> it seems that GFp_rsa_check_modulus_and_exponent()should check that e is
> not more than 33 bits, which is actually what it does, with e_bits >
> kMaxExponentBits. However, 2 ** 34 - 1 has a length of 34 bits. I've
> therefore used 2 ** 33 +/- 1 in 1f450f9
> <https://github.com/briansmith/ring/commit/1f450f9fbe2f69becfe31a14d96b734a9aafa6a8>.
> If you think that's okay, I'll look into the zero padding next.
>
Yes, do what I mean, not what I say! :) The important thing is to test that
bounds check.

Cheers,
Brian
-- 
https://briansmith.org/